src/types.ts

```diff
@@ -55,4 +56,9 @@ export interface Props {
   schema?: GraphQLSchema
   context?: Context | ContextCallback
   options?: Options
+  formatError?: Function
+  formatParams?: Function
+  formatResponse?: Function
+  logFunction?: LogFunction
+  debug?: boolean
```

Comment:
Where is debug being used?

Reply:
Oops.. where did that come from.. Anyways I've removed it.